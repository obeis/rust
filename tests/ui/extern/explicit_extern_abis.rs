//@ compile-flags: -Z unstable-options --edition 2027

extern fn foo() {}
//~^ ERROR extern declarations without an explicit ABI are disallowed

fn main() {}
