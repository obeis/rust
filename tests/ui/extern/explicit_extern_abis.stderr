error: extern declarations without an explicit ABI are disallowed
  --> $DIR/explicit_extern_abis.rs:3:1
   |
LL | extern fn foo() {}
   | ^^^^^^

error: aborting due to 1 previous error

